server/room.mjs

```diff
@@ -6,6 +6,7 @@ import FileLoader from './fileloader.mjs';
 import FileUpdater from './fileupdater.mjs';
 import Logging from './logging.mjs';
 import Config from './config.mjs';
+import { randomHue } from '../client/js/color.js';
```

Comment:
Should we put the file into a different folder if we use it in the server _and_ the client? Or should we do something else like just adding a comment at the top of `color.js`? Or just leave it as is? I don't really mind.